.github/workflows/publish-operator-opamp-bridge.yaml

```diff
@@ -0,0 +1,70 @@
+name: "Publish Operator OpAMP Bridge"
```

Comment:
Note: if we start adding more containers that get built in this repo, we should probably make a single action to publish multiple containers.